forest/sycamore/functions.py

```diff
@@ -209,35 +268,22 @@ def parse_surveys(config_path, answers_l=False):
                 if 'text_field_type' in q.keys():
                     surv['text_field_type'] = q['text_field_type']
                 # Convert surv to data frame
-                #                 surv = pd.DataFrame([surv]).merge(timings, left_on = 'config_id', right_on = 'config_id')
 
                 if answers_l:
                     if 'answers' in q.keys():
-                        for i, a in enumerate(q['answers']):
+                        for j, a in enumerate(q['answers']):
                             surv['answer_' + str(i)] = a['text']
 
                 output.append(pd.DataFrame([surv]))
     output = pd.concat(output).reset_index(drop=True)
     return output
 
 
-def convert_timezone(utc_date, tz_str):
-    """
-    Args:
-        utc_date(datetime):
-            Given date, assumed to be in UTC time
-        tz_str(str):
-            timezone to convert the utc_date too (a string used by the pytz library)
-
-    Returns:
-        A converted date from UTC time to the given timezone
-    """
-    date_local = pytz.utc.localize(utc_date, is_dst=None).astimezone(tz_str)
-    return date_local
-
+def convert_timezone_df(df_merged: pd.DataFrame,
+                        tz_str: str = "UTC",
+                        utc_col: str = 'UTC time') -> pd.DataFrame:
```

Comment:
"Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability."
https://www.python.org/dev/peps/pep-0008/#string-quotes

I know this hasn't been very consistent throughout the project so far but let's stick with double quotes as the default.